tests/foreman/ui/test_contentview.py

```diff
@@ -702,6 +702,108 @@ def test_negative_add_components_to_non_composite(self):
 
     @run_only_on('sat')
     @tier2
+    def test_negative_add_unpublished_cv_to_composite(self):
```

Comment:
Ah, now I see. That's different cases.
1. `add_components_to_` `non_composite` tries to add simple CV to another simple CV and checks that it is not possible at all, i.e. `ContentView` tab is not there.
2. `add_unpublished_cv_to_` `composite` tries to add simple CVs to to composite CV and checks that it is possible to add only published CV.